scripts/processing/01-run_anatomy.ipy

```diff
@@ -2,12 +2,12 @@
 Run Freesurfer/MNE anatomical pipeline
 ======================================
 
-This runs Freesurfer recon-all on all subjects
-and computes the BEM surfaces later used for forward
-modeling. BEM extraction is done using flash MRI
-data.
-"""
+This runs Freesurfer recon-all on all subjects and computes the BEM surfaces
+later used for forward modeling. BEM extraction is done using flash MRI data.
 
+Make sure to run `source SetUpFreeSurfer.sh` (bash and derivatives) or `source
+SetUpFreeSurfer.csh` (csh and derivatives) before running this script.
```

Comment:
umm ... isn't this part of the Freesurfer installation instructions: https://surfer.nmr.mgh.harvard.edu/fswiki/LinuxInstall? I'm a bit surprised that you missed this